docs/standards/smart-contracts/lsp0-erc725-account.md

```diff
@@ -53,7 +53,7 @@ Sets the **initial owner** of the contract and registers **[LSP0ERC725Account](.
 function owner() public view returns (address owner)
 ```
 
-Returns the address of the current owner.
+The function returns the address of the current owner of the smart contract.
```

Comment:
The sentence descriptions in the contracts API docs need to be very short.
See Uniswap Contract API docs as an example for the sentence: https://docs.uniswap.org/protocol/reference/core/UniswapV3Factory#setowner

```suggestion
Returns the address of the current account owner.
```